atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -1149,26 +1183,67 @@ private int estimateSize(List<RowResult<Value>> rangeRows) {
 
         getMeter(AtlasDbMetricNames.SNAPSHOT_TRANSACTION_CELLS_READ, tableRef).mark(rawResults.size());
 
+        Collection<Map.Entry<Cell, T>> resultsAccumulator = new LinkedList<>();
```

Comment:
In general, when instantiating a *specific* type, would be good to leave a comment on why that type was chosen.

Reply:
The added comment should be sufficient.